test/k8s-integration/main.go

```diff
@@ -391,6 +391,7 @@ func generateGCETestSkip(clusterVersion, platform string) string {
 	// bug-fix introduced in 1.16
 	if v.LessThan(apimachineryversion.MustParseSemantic("1.16.0")) {
 		skipString = skipString + "|volumeMode\\sshould\\snot\\smount\\s/\\smap\\sunused\\svolumes\\sin\\sa\\spod"
+		skipString = skipString + "|allowExpansion"
```

Comment:
Added this w.r.t pre-requiste for enabling 1.15 test jobs.
I see readme suggests we support resizer beta from 1.16 onwards.